src/Aspire.Dashboard/Components/App.razor

```diff
@@ -14,7 +14,22 @@
     <!-- Make the body background dark if dark mode will be enabled to prevent a flash of white
          before fully rendered. This value is from --neutral-fill-layer-rest, but that custom
          property isn't available at this point. -->
-    <style>@@media (prefers-color-scheme: dark) { body { background-color: #272727; } }</style>
+    <style>
+        @@media (prefers-color-scheme: dark) {
+            body {
+                background-color: #272727;
+            }
+        }
+
+        #components-reconnect-modal :is(h5, p) {
```

Comment:
What's the reasoning behind putting these styles directly in App.razor as opposed to app.css, especially since they use vars that aren't defined until app.css is loaded?

Reply:
Really, it was just because of the other key set directly in App.razor, but I moved to app.css